### PR TITLE
Related items skeleton

### PR DESCRIPTION
**Pull request by charliebailey24**

### CODE REVIEW

**Comment by charliebailey24** [comment]
quick fix

**Comment by charliebailey24** [comment]
If anyone wants to meet their PR req, please feel free to rip me a new one on this PR.

File: src/components/relatedItems/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import axios from 'axios';
+
+class Card extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```
Comment (jerremy777) [inline src/components/relatedItems/Card.jsx:7]: Why so much state? Some of these never change so might be able to just be props?

**Review by jerremy777 — APPROVED** [review]

File: src/components/relatedItems/Card.jsx (hunk 1 above)
Comment (jerremy777) [reply src/components/relatedItems/Card.jsx:7]: Also add a key prop to your list of cards. Its super easy to do and makes it so you don't have to keep seeing the error come up in console.

Comment (charliebailey24) [reply src/components/relatedItems/Card.jsx:7]: So my cardList component will receive the current productId from App, and then will get the related items array for that productId and create a card for each related item productId. So right now, the only thing being passed into each card is the productId from the related items API call, and everything in state is what I'm getting in my getAllInfo function, which I'm then storing in state to update the card. Good point though that I could probably move all of this logic up to cardList and just pass that info down as props. I'll take a look into that.

**Pull request merged**